soh/src/code/z_camera.c

```diff
@@ -1677,10 +1677,12 @@ s32 Camera_Normal1(Camera* camera) {
         Camera_ClampDist(camera, eyeAdjustment.r, norm1->distMin, norm1->distMax, anim->unk_28);
 
     if (anim->startSwingTimer <= 0) {
+        if (CVarGetInteger("gA11yDisableIdleCam", 0)) return;
         eyeAdjustment.pitch = atEyeNextGeo.pitch;
         eyeAdjustment.yaw =
             Camera_LERPCeilS(anim->swingYawTarget, atEyeNextGeo.yaw, 1.0f / camera->yawUpdateRateInv, 0xA);
     } else if (anim->swing.unk_18 != 0) {
+        if (CVarGetInteger("gA11yDisableIdleCam", 0)) return;
```

Comment:
it's not clear why it's only these 2 places where we want to early return (instead of also in the `else`), maybe adding a comment explaining would be helpful?